apps/docs/src/components/Menu/LeftSidebar/LeftSidebarCategory.svelte

```diff
@@ -36,16 +28,15 @@
           {item.title}
         </div>
       {:else}
-        <li class="last:mb-0">
-          <a
-            class={[
-              'group relative block border-l-2 py-1 pl-4 pr-4 hover:text-white',
-              isEqual(activeUrlPathName, `${category.urlPrefix}/${item.slug}`)
-                ? 'border-orange font-bold text-orange'
-                : 'border-white/20 hover:border-white'
-            ]}
-            href={`${category.urlPrefix}/${item.slug}`}
-          >
+        <li
+          class={[
+            'sidebar-list-item',
+            activeUrlPathName === `${category.urlPrefix}/${item.slug}`
```

Comment:
We don't need to account for starting & trailing slashes here?

Reply:
I'd leave it in, it's probably safer if someone navigates from an external source that adds a slash somewhere